### DISCUSSION TITLE
How to do embeddings

### DISCUSSION THREAD

**Discussion by rapidarchitect** [abdeladim-s/pyllamacpp#31] [General]
I cant find any documents on how to do embeddings in python vs command line with this library.. need a lil direction please :)

**Comment by absadiki** [comment]
@rapidarchitect, the command line is just a simple chat interface.
To get the embeddings, you can do it easily from Python by calling [get_embeddings](https://abdeladim-s.github.io/pyllamacpp/#pyllamacpp.model.Model.get_embeddings) from a model instance. 
Hope this is what you are looking for ?
